internal/bindings/build.go

```diff
@@ -1,6 +1,6 @@
 package bindings
 
 /*
-#cgo linux LDFLAGS: -lraft -ldqlite -Wl,-z,now
+#cgo linux LDFLAGS: -ldqlite -lraft -llz4 -luv_a -lm -Wl,-z,now
```

Comment:
Not really. It appears that my distribution (ubuntu 22.04) comes with the `libuv_a.a` file instead of libuv.a, and I assumed that this could be the default naming convention for the static version of the libuv library.

However, the switch for static linking comes with the -static ld flag, which forces ld to look for *.a files instead of *.so files. I initially thought that this wouldn't affect dynamic linking at all since the .a extension would be ignored during dynamic linking. At least it seems like that's what's happening on my system. But yeah, it looks a little dirty. 

Didn't know that you already had plans to support fully-static builds. That will help a lot, thank you!